pam/main-cli.go

```diff
@@ -0,0 +1,45 @@
+//go:build !asPamModule
```

Comment:
That' tag is defined inside the `pam-module.go` so that's what generated when building the module as such.

However it's also possible to generate a `pam` binary that is used for testing purposes. The two have different scope and they must include different symbols (compare with `m -gD`) that I don't want them to be exposed when building the library.

For the way upstream code is made that was the behavior, I can likely change it including a custom tag in our project.

Reply:
Switching to `pam_binary_cli` tag.